carbon/app.py

```diff
@@ -38,7 +38,9 @@ class CarbonFtpsTls(FTP_TLS):
         See ftplib.FTP_TLS for more details.
     """
 
-    def ntransfercmd(self, cmd: str, rest: str | int | None = None) -> tuple[socket, int]:
+    def ntransfercmd(
+        self, cmd: str, rest: str | int | None = None
+    ) -> tuple[socket, int | None]:
```

Comment:
I was confused by this at first but it looks like the type hint was indeed updated, see #4 [here](https://github.com/python/typeshed/issues/10466)

Reply:
Nice find!  Helpful to understand why it changed.